src/app/containers/Ad/Amp/index.jsx

```diff
@@ -36,29 +63,54 @@ const ampAdPropsDesktop = ({ service }) => ({
   json: JSON.stringify(constructAdJsonData({ service })),
 });
 
+const AMP_ACCESS_DATA = (endpoint) => ({
+  authorization: endpoint,
+  noPingback: true,
```

Comment:
I don't think we need a pingback to happen so I've disabled it
context: https://amp.dev/documentation/components/amp-access/#pingback-endpoint